ibridges/data_operations.py

```diff
@@ -55,10 +55,10 @@ def upload(
         Absolute irods destination path
     overwrite:
         If data object or collection already exists on iRODS, overwrite
-    ignore_err:
-        If an error occurs during upload, and ignore_err is set to True, any errors encountered
-        will be transformed into warnings and iBridges will continue to upload the remaining files.
-        By default all errors will stop the process of uploading.
+    on_err:
+        'fail': Stop transfer and throw exception
```

Comment:
This looks kind of bad in the API reference, since new lines are not preserved. I would put this more in a normal sentence. Like "Decides what happens when an error occurs. There are three options: ..."